lms/djangoapps/bulk_email/tasks.py

```diff
@@ -418,7 +418,22 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
         'failed' count above.
     """
```

Comment:
If I wanted to pull all lines related to this in SPLUNK will it be easy to do so? I don't quite see how but trust ur judgement.